x/erc20/genesis_test.go

```diff
@@ -71,41 +81,61 @@ func (suite *GenesisTestSuite) TestERC20InitGenesis() {
 	testCases := []struct {
 		name         string
 		genesisState types.GenesisState
+		malleate     func()
 	}{
 		{
 			"empty genesis",
 			types.GenesisState{},
+			nil,
```

Comment:
nit: I think it's more verbose to add the field names and omit the empty fields for all of these:

```suggestion
			name: "empty genesis",
			genesisState: types.GenesisState{},
```